client/flutter/lib/api/notifications.dart

```diff
@@ -19,8 +19,10 @@ class Notifications {
         await NotificationPermissions.getNotificationPermissionStatus();
     var granted;
     switch (permissionStatus) {
+      case PermissionStatus.provisional:
```

Comment:
[The iOS Permissions Docs](https://developer.apple.com/documentation/usernotifications/asking_permission_to_use_notifications
) show that provisional permissions do send notifications to the center.
```suggestion
      case PermissionStatus.provisional:
        granted = true;
```